DuckDuckGo/UnifiedFeedbackForm/UnifiedFeedbackFormViewModel.swift

```diff
@@ -51,6 +62,26 @@ final class UnifiedFeedbackFormViewModel: ObservableObject {
         case reportFAQClick
     }
 
+    struct Payload: Codable {
+        let userEmail: String
+        let feedbackSource: String
+        let platform: String
+        let problemCategory: String
+
+        let feedbackText: String
+        let problemSubCategory: String
+        let customMetadata: String
+
+        func toData() -> Data? {
+            try? JSONEncoder().encode(self)
+        }
```

Comment:
This logic could benefit from a unit testing pass

Reply:
Actually I think this is fine after looking at the test file more - deleting this comment